src/core/lwip/tcp.c

```diff
@@ -905,6 +905,11 @@ err_t tcp_recv_null(void *arg, struct tcp_pcb *pcb, struct pbuf *p, err_t err)
     return ERR_OK;
 }
 
+static inline u32_t get_initial_rto(void)
+{
+    return (1000 + slow_tmr_interval - 1) / slow_tmr_interval;
```

Comment:
Until we improve timer precision, the number of ticks needs to be > 1. Otherwise, RTO can be triggered immediately mistakenly.